modules/cloudtrail/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "company_id" {
+  type = string
+}
+
+variable "lambda_debug" {
+  type    = bool
+  default = false
+}
+
+variable "lambda_timeout" {
+  type    = string
+  default = "30s"
+}
+
+variable "ingest_url" {
+  type    = string
+  default = "https://api.ksoc.com"
```

Comment:
Might want to avoid a default value here just in case they accidentally send data to prd when they don't wan to